requirements-dev.txt

```diff
@@ -167,7 +167,7 @@ flipper-client==1.3.2
     # via
     #   -r requirements.txt
     #   funding-service-design-utils
-funding-service-design-utils==1.0.28
+funding-service-design-utils @ git+https://github.com/communitiesuk/funding-service-design-utils.git@fund-store-db
```

Comment:
This feels a temporary change that shouldn't be commited

Reply:
I can't then resolve this until that PR is merged. For records sake: https://github.com/communitiesuk/funding-service-design-utils/pull/74 is the PR